src/main/riddl/Tenant/tenantMessages.riddl

```diff
@@ -65,15 +65,15 @@ event TenantNameChanged is { tenantId: TenantId, oldName: String, newName: Strin
 // command SwapDistributionAccount is { tenantId: TenantId, newDistributionAccount: DistributionAccountId, updatingUser: String }
 // event DistributionAccountSwapped is { tenantId: TenantId, name: String, oldAccount: DistributionAccountId, newAccount: DistributionAccountId, meta: TenantContext.MetaInfo }
 
-query FindTenantsByMember is {member: MemberId} briefly "Get a list of organizations that a member belongs to."
+query FindTenantsByMember is {member: MemberId} briefly "Get a list of tenants that a member belongs to."
 result TenantListByMember is {
   member: MemberId,
-  organizations: TenantId*
+  tenants: TenantId*
 }
-query FindTenantsByOrganization is {owner: MemberId} briefly "Get a list of organizations that are owned by a member."
-result TenantListByOrganization is {
-  organizations: OrganizationId,
-  owner: TenantId*
+query FindTenantsByOwner is {owner: MemberId} briefly "Get a list of tenants that are owned by a member."
```

Comment:
I think it will be rare that a member will own multiple tenants. Actually, as I look at the definition of Tenant there is a PrimaryContact (not a Member), and there is not a concept of Owner. So I am not sure what this query is about. As I look at the deleted lines it may be something I had put in there, and if it was me I am not sure what I was thinking at the time. For now, at least, I don't think we need this query.